app/connector/DestinationConnector.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connector
+
+import config.AppConfig
+import javax.inject.Inject
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DestinationConnector @Inject()(val config: AppConfig, val http: HttpClient)(implicit ec: ExecutionContext) {
+
+  def sendMessage(responseData: String, headers: Seq[(String, String)])(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+
+    val serviceUrl = s"${config.traderAtDestinationUrl}/message"
```

Comment:
Could the _message_ be part of the traderAtDestinationUrl?

Reply:
@chris-ianson later in future if we need to call some other end point in trader-at-destination
for flexibility better to keep traderAtDestinationUrl as it is rather then binding specific to message url

Reply:
serviceUrl is only used in one place. The url should include message and if we need to change it in future we only have to change it in AppConfig